pyspecdata/figlist.py

```diff
@@ -64,7 +64,7 @@
 from .plot_funcs import image as this_image
 from .general_functions import *
 from . import plot_funcs as this_plotting
-from .core import nddata, autolegend, gridandtick
+from .core import nddata, autolegend, gridandtick, r_
```

Comment:
just a minor import I know it's unrelated but I was getting an error when plotting that this fixed

Reply:
from numpy, not .core

Reply:
okay added question on slack!